db/schema.rb

```diff
@@ -46,6 +46,34 @@
     t.index ["reset_password_token"], name: "index_admin_users_on_reset_password_token", unique: true
   end
 
+  create_table "book_sugestions", force: :cascade do |t|
+    t.string "editorial"
```

Comment:
Hmm your table is 2 times here, maybe the `g` issue? if that's the case remove the schema.rb file and regenerate it

Reply:
got it